tests/__tests__/mixins/repetition.js

```diff
@@ -0,0 +1,40 @@
+import { getWaveInstance } from "../../enums";
+import { takeSnapshotAndAssertEqualityAsync } from "../../utils";
+
+describe('Screen tests for axes', () => {
+    const settings = {
+        atomRadiiScale: 0.2,
+        XRepetitions: 1,
+        YRepetitions: 1,
+        ZRepetitions: 1,
+    }
+
+    test('X axis', () => {
```

Comment:
Test names are a bit weird. Maybe "Repeat along lattice vector A" here, and "Repeat along lattice vectors A and B" instead of "XY axis below - and so on?